.github/workflows/ci.yml

```diff
@@ -43,6 +43,7 @@ jobs:
 
       - name: Run miri
         run: |
+          set -euxo pipefail
```

Comment:
I'm assuming this was for debugging?

Reply:
Not even, I just added it as a proactive measure. Otherwise you can't see in the logs when each line is run.